aula/apps/presencia/templates/presencia/winwheel.html

```diff
@@ -0,0 +1,87 @@
+{% extends "base.html" %}
+ 
+{% block extrahead %}
+  <script type="text/javascript" src="{{STATIC_URL}}Winwheel/Winwheel.min.js"></script>
+  <script type="text/javascript" src="{{STATIC_URL}}Winwheel/TweenMax.min.js"></script>
+  <style>
+    td.the_wheel
+      {
+          background-image: url("{{STATIC_URL}}Winwheel/wheel_back.png");
+          background-position: center;
+          background-repeat: no-repeat;
+      }
+  </style>
+{% endblock %}
+
+ 
+{% block content  %}
+{%if hi_ha_prous_alumnes %}
+
+<table cellpadding="0" cellspacing="0" border="0">
+  <tr>
+      <td width="438" height="582" class="the_wheel" align="center" valign="center">
+          <canvas id='canvas' width='434' height='434'>
+            El guanyador és {{guanyador_no_ui}}
+          </canvas>
+      </td>
+  </tr>
+</table>
+
+
+  {% autoescape off %}
+  <script>
+    let wheelSpinning = false;
+    let theWheel = new Winwheel({
+    'numSegments'    : {{n}},
+    'segments'       : {{items_ruleta}},
+    'animation' :
+      {
+          'type'     : 'spinToStop',
+          'duration' : 20,
+          'spins'    : 4,
+          'callbackSound'    : playSound,   
+          'callbackFinished' : whellstops,
+          'soundTrigger'     : 'segment'        // Specify pins are to trigger the sound, the other option is 'segment'.      }
+        }
+    });    
+  {% endautoescape %}
+
+  let audio = new Audio('{{STATIC_URL}}Winwheel/tick.mp3');  // Create audio object and load tick.mp3 file.
+
+  function whellstops(i)
+  {
+    wheelSpinning = false;
+  }
+  function playSound()
+  {
+      // Stop and rewind the sound if it already happens to be playing.
+      audio.pause();
+      audio.currentTime = 0;
+
+      // Play the sound.
+      audio.play();
+  }
+
+
```

Comment:
Crec que li falta una mica de random. Fent proves consecutives, prement "Fer rodar", acostumen a sortir els mateixos noms.
```suggestion
  function getRandomInt(min, max) {
      return Math.floor(Math.random() * (max - min)) + min;
  }

  function startSpin()
  {
      duration = getRandomInt(15,20);
      spins = (duration+2)/5;
      if (!wheelSpinning) {
          theWheel.stopAnimation(false);  // Stop the animation, false as param so does not call callback function.
          theWheel.rotationAngle = 0;     // Re-set the wheel angle to 0 degrees.
          theWheel.animation.duration = duration;
          theWheel.animation.spins = spins;
          theWheel.draw();                // Call draw to render changes to the wheel.        
          theWheel.startAnimation();
          wheelSpinning = true;
      }
  }
```

Reply:
He aplicat aquest codi a la versió final. He aprofitat també per:
* Fer disable del botó que fa rodar la winwheel mentre aquesta està rodant per tornar a activar-lo en finalitzar.
* Treure els comentaris JS.

@amorilla, et torno a demanar validació i, si et sembla tot bé, tu mateix fes `Squash and merge` + `delete branch`. Si veus qualsevol cosa a millorable tot comentari és benvingut. Merci per la review.